pages/api/dpu/aliases.ts

```diff
@@ -27,4 +40,30 @@ const aliasesHandler = async (req: NextApiRequest, res: NextApiResponse) => {
     });
 };
 
-export default aliasesHandler;
+const aliasesGetHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+    console.info("[aliasesGetHandler] Retrieving aliases from DB...");
+
+    // Extract query parameters
+    const { repo_name, repo_owner, repo_provider } = req.query;
+
+    // Validate query parameters
+    if (!repo_name || !repo_owner || !repo_provider) {
+        console.error("[aliasesGetHandler] Missing query parameters");
+        res.status(400).json({ "error": "Missing query parameters" });
+        return;
+    }
+
+    // Retrieve aliases from the database
+    try {
+        const aliases = await getUserAliasesFromDb(
+            repo_name as string, repo_owner as string, repo_provider as string
+        );
+        console.info("[aliasesGetHandler] Aliases retrieved successfully");
+        res.status(200).json({ aliases });
+    } catch (error) {
```

Comment:
remove try-catch and use .catch on await statement

Reply:
done